aws-iotfleethub-application/inputs/inputs_1_create.json

```diff
@@ -1,5 +1,11 @@
 {
```

Comment:
change looks good. Need to fix build failure tho

Reply:
Fixed